sym_api_client_python/services/datafeed_id_repository.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import os
+
+DATAFEED_ID_FILE = 'datafeed.id'
+
+log = logging.getLogger(__name__)
+
+
+class OnDiskDatafeedIdRepository:
+    def __init__(self, datafeed_id_folder):
+        self.datafeed_id_file_path = OnDiskDatafeedIdRepository._get_datafeed_id_file_path(datafeed_id_folder)
+
+    def read_datafeed_id_from_file(self):
+        log.debug(f'Retrieving datafeed id from file {self.datafeed_id_file_path}')
+        if os.path.exists(self.datafeed_id_file_path):
+            with open(self.datafeed_id_file_path, 'r') as datafeed_id_file:
+                line = datafeed_id_file.readline()
+                if line:
+                    index = line.find("@")
+                    if index != -1:
+                        datafeed_id = line[0:index]
+                        log.debug(f'Retrieved datafeed id: {datafeed_id}')
+                        return datafeed_id
+        log.debug(f'Could not retrieve datafeed id from file {self.datafeed_id_file_path}')
+        return None
+
+    def store_datafeed_id_to_file(self, datafeed_id, agent_url):
+        with open(self._get_datafeed_id_file_path(), 'w') as datafeed_id_file:
+            line = f'{datafeed_id}@{agent_url}'
+            log.debug(f'Writing {line} to {self.datafeed_id_file_path}')
+            datafeed_id_file.write(line)
+
+    @staticmethod
+    def _get_datafeed_id_file_path(datafeed_id_folder):
```

Comment:
I don't think this method should be a static method.

Reply:
I put it based on my IDE's advice (`self` is not used). I can put it as member function.